.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,6 +4,8 @@
 
 <!-- Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. -->
 
+<!-- Does this change require an update to documentation? -->
```

Comment:
```suggestion
- [ ] This change requires an update to documentation.
```

Reply:
This was agreed to be a comment. I'm happy to change it but can you sign off @davidnuzik

Reply:
I vote we just make it a comment. I like the idea of the checkbox -- but to be honest no one seems to use them. I use them in my gh issues for releases and no one checks them off as we go. I say go with the HTML comment for now. If it doesn't seem to be helping we can revisit this and modify as needed later.